specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json

```
@@ -5236,7 +5236,7 @@
         "value": {
           "x-ms-client-name": "base64EncodedCertificate",
           "type": "string",
-          "description": "Base64 encoded representation of the certificate object to import. This certificate needs to contain the private key."
+          "description": "A PEM file or a Base64 encoded PFX file.  PEM files need to contain the private key."
```

"base64-encoded" is a compound adjective, and "base64" is not a proper noun.
```suggestion
          "description": "A PEM file or a base64-encoded PFX file.  PEM files need to contain the private key."
```